src/storage/mongodb/LoggingStorage.ts

```diff
@@ -80,6 +81,7 @@ export default class LoggingStorage {
     // Insert
     if (global.database) {
       await global.database.getCollection<Log>(tenantID, 'logs').insertOne(logMDB);
+      return logMDB._id;
```

Comment:
Could you please add a comment on the reason with pushConvertObjectIDToString() helper is not here? 
`// For performance reason, the ObjectID type is returned and used.`
Or something similar.

Reply:
I checked most of the project code and noticed that `pushConvertObjectIDToString()` is only used in the **get** methods, where an aggregation is built. 
This is the `saveLog()` method and I noticed that for the **save** methods that return an id, this is being done as I initially wrote it - `return logMDB._id.toHexString()` in this case. (You can check saveSite, saveAsset, saveInvoice etc. ; they all use `.toHexString()` on the _id for the return value.) 
Is it ok if I revert to this approach here too, since it is how the code is wrote in most of the other save methods? I need the id as a string anyway so I am doing this conversion sooner or later.
Or is this a special case?

Reply:
Fine, keep it as it's done currently in that PR. 

The ObjectID conversion to string in ev-server should be audit later on because: 
+ information are lost and it made debugging a lot less efficient: https://docs.mongodb.com/manual/reference/method/ObjectId/
+ the conversion is costly and should only be done if needed to avoid a global information loss on the object instance.
That could be part of another issue and PR.

Reply:
Understood, thanks.